### PR TITLE
Putting original authors back in

### PR DESCRIPTION
**Pull request by philarcher**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/pull/74.html" title="Last updated on Apr 2, 2020, 4:13 PM UTC (799bed7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/74/183f671...799bed7.html" title="Last updated on Apr 2, 2020, 4:13 PM UTC (799bed7)">Diff</a>

### CODE REVIEW

**Comment by philarcher** [comment]
I've just added Kim, Ryan and Adrian to the list of Authors. 
@iherman can you please add their W3C IDs? (or show me how I can do it as a lowly AC rep - boy I miss team access!)

File: index.html [hunk 1]
```diff
@@ -52,6 +52,12 @@
           company: "GS1",
           w3cid: 101627
         }],
+	authors: [{
+		name: "Kim Hamilton-Duffy",
+		url: "https://www.okimsrazor.com/"
+	},
```
Comment (iherman) [inline index.html:59]: w3cid: 99918

File: index.html [hunk 2]
```diff
@@ -52,6 +52,12 @@
...
+		  {name: "Ryan Grant"},
```
Comment (iherman) [inline index.html]: w3cid: 101398

File: index.html [hunk 3]
```diff
@@ -52,6 +52,12 @@
...
+		  {name: "Adrian Gropper"}],
```
Comment (iherman) [inline index.html]: w3cid: 91611

**Review by iherman — APPROVED** [review]

**Comment by iherman** [comment]
@philarcher I cannot modify the file itself, it is under your control on your repo... but you can find the ids above.

**Comment by philarcher** [comment]
Thanks @iherman. I've added those IDs into the doc.

**Review by jandrieu — APPROVED** [review]

**Comment by jandrieu** [comment]
Looks good.

**Pull request merged**